Classes/Updates/FieldLogoMigrationWizard.php

```diff
@@ -0,0 +1,230 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the package jweiland/daycarecenters.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace JWeiland\Daycarecenters\Updates;
+
+use Doctrine\DBAL\DBALException;
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+use TYPO3\CMS\Core\Core\Environment;
+use TYPO3\CMS\Core\Database\ConnectionPool;
+use TYPO3\CMS\Core\Database\Query\Expression\ExpressionBuilder;
+use TYPO3\CMS\Core\Resource\FileInterface;
+use TYPO3\CMS\Core\Resource\ResourceStorage;
+use TYPO3\CMS\Core\Resource\StorageRepository;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Install\Attribute\UpgradeWizard;
+use TYPO3\CMS\Install\Updates\UpgradeWizardInterface;
+
+#[UpgradeWizard('daycarecentersFieldLogoMigration')]
+class FieldLogoMigrationWizard implements UpgradeWizardInterface, LoggerAwareInterface
+{
+    use LoggerAwareTrait;
+
+    public function getTitle(): string
+    {
+        return '[daycarecenters] Migrate FAL relations where field name with logo to logos';
```

Comment:
This migrator is a some kind of miss-match between migrating an image path to FAL AND changing logo to logos.